src/explore.c

```diff
@@ -133,7 +220,10 @@ void dump_state(sqlite3 *db, sds *current_line, int *current_selection) {
             *current_selection = 0;
         }
         for(size_t f = 0; f < SEARCH_LIMIT; f++) {
-            if(explore_total > 0 && f == *current_selection) {
+            if(explore_total > 0 && hitsannotations[f] == 1) {
+                printf(PRETTY_PRUNE "%s" PRETTY_NORM CLEAR_LINE "\n", hits[f]);
+            }
```

Comment:
Print out the line using the prune color if it is currently marked for pruning (annotation == `1`)